wrangler-proto/src/main/java/io/cdap/wrangler/proto/adls/FileQueryDetails.java

```diff
@@ -0,0 +1,59 @@
+/*
+ * Copyright © 2018 Cask Data, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License"); you may not
+ * use this file except in compliance with the License. You may obtain a copy of
+ * the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations under
+ * the License.
+ */
+package io.cdap.wrangler.proto.adls;
+
+public class FileQueryDetails {
+
+    private String filePath;
+    private int lines;
+    private String sampler;
+    private double fraction;
+    private String scope;
+    private String header;
+
+    public FileQueryDetails(String header, String filePath, int lines, String sampler, double fraction, String scope) {
+        this.filePath = filePath;
```

Comment:
We use two spaces instead of four as the coding convention. Please update all files to honor that.